runtime/ft_test.go

```diff
@@ -371,15 +371,15 @@ access(all) contract FlowToken: FungibleToken {
         // Create a public capability to the stored Vault that only exposes
         // the 'deposit' method through the 'Receiver' interface
         //
-        adminAccount.link<&FlowToken.Vault{FungibleToken.Receiver}>(
+        adminAccount.link<&FlowToken.Vault>(
```

Comment:
This doesn't matter anymore with entitlements; you can no longer "expose the whole `Vault`" because the reference here is still unauthorized. This is the basis for the planned migration to remove restricted types; any types of the form `T{I}` are just going to be replaced with `T` because it's no longer unsafe to give out a reference to the whole type as long as you use proper entitlements.

Reply:
ah ok, makes sense!